### PR TITLE
fix: portfolio table sorting

### PR DESCRIPTION
**Pull request by alter-eggo**
This pr fixes portfolio table sorting, mentioned [here](https://discord.com/channels/638460494168064021/1317147929374490685/1317147992234397789)

### CODE REVIEW

**Comment by garethfuller** [comment]
@alter-eggo Can you merge in main and fix the PR please.

File: packages/lib/modules/pool/PoolDetail/PoolActivityTable/PoolActivityTable.tsx [hunk 1]
```diff
@@ -31,18 +31,14 @@ export function PoolActivityTable() {
...
-  }))
-
   return (
     <>
       <Box className="hide-scrollbar" overflowX="auto" w="full">
         <Box minWidth="800px">
           <PaginatedTable
             alignItems="flex-start"
-            items={items}
+            getRowId={([_timestamp, _value, poolActivityMetaData]) => poolActivityMetaData.id}
```
Comment (agualis) [inline packages/lib/modules/pool/PoolDetail/PoolActivityTable/PoolActivityTable.tsx:40]: Lint problems here

File: packages/lib/modules/pool/PoolList/PoolListTable/PoolListTable.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ export function PoolListTable({ pools, count, loading }: Props) {
     >
       <PaginatedTable
         items={pools}
+        getRowId={item => item.id}
```
Comment (agualis) [inline packages/lib/modules/pool/PoolList/PoolListTable/PoolListTable.tsx:50]: More lint issues

File: packages/lib/shared/components/tables/PaginatedTable.tsx [hunk 3]
```diff
@@ -10,17 +10,19 @@ interface Props<T> extends BoxProps {
   showPagination: boolean
   paginationProps: any // TODO: type this
   noItemsFoundLabel: string
+  getRowId: (item: T, index: number) => React.Key
 }
 
-export function PaginatedTable({
+export function PaginatedTable<T extends any>({
```
Comment (agualis) [inline packages/lib/shared/components/tables/PaginatedTable.tsx:16]: More lint issues

**Review by agualis — CHANGES_REQUESTED** [review]

**Review by agualis — APPROVED** [review]

**Pull request merged**